cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -25,11 +25,16 @@
 #include "Eigen/Geometry"
 #include "OgreGpuProgramParams.h"
 #include "OgreImage.h"
+#include "cartographer/common/make_unique.h"
 #include "cartographer/common/port.h"
 #include "cartographer_ros_msgs/SubmapQuery.h"
 #include "eigen_conversions/eigen_msg.h"
 #include "ros/ros.h"
 
+#if (QT_VERSION < QT_VERSION_CHECK(5, 0, 0))
```

Comment:
Can we use QString instead and format at runtime? I'll take the minimal performance hit in rVize any day over simpler code.